src/Smuggler/Plugin.hs

```diff
@@ -121,3 +119,6 @@ unusedLocs (L (RealSrcSpan loc) decl, used, unused)
         L _ (IEName (L (RealSrcSpan lieLoc) name)) <- [lieName]
         guard $ name `elem` unused
         pure (srcSpanStartLine lieLoc, srcSpanStartCol lieLoc)
+    getEndColMax u = maximum $ map (findColLoc . nameSrcSpan) u
+    findColLoc (RealSrcSpan l) =  srcSpanEndCol l
+    findColLoc (UnhelpfulSpan _) = 100
```

Comment:
Could you, pleaase, add types to functions inside `where`? This is very helpful for code understanding and maintaing, especially when working with complicated frameworks like GHC

Reply:
That is precisely what I argue about in work but my colleagues seem to think that type annotations are not so important inside of where. You are completely right about this.